webui/src/lib/components/repository/changes.jsx

```diff
@@ -203,7 +203,7 @@ export const ChangesTreeContainer = ({results, showExperimentalDeltaDiffButton =
                                     <ArrowLeftIcon/> Back to object comparison
                                   </Button>
                                 : <div className="mr-1 mb-2"><Alert variant={"info"}><InfoIcon/> You can now use lakeFS to
```

Comment:
```suggestion
                                : <div className="mr-1 mb-2"><Alert variant={"info"}><InfoIcon/> You can use lakeFS to
```

per above note on use of "Now"

Reply:
done